phantombot/umbrel-app.yml

```diff
@@ -0,0 +1,36 @@
+manifestVersion: 1.1
+id: phantombot
+category: social
+name: PhantomBot
+version: "3.14.1.0"
+tagline: PhantomBot is a Twitch chat bot powered by Java
+icon: ""
```

Comment:
This entire `icon` line can be removed. We'll host the icon here: https://github.com/getumbrel/umbrel-apps-gallery

And the icon rendering logic in umbrelOS will grab the correct icon.